engine/installation/linux/fedora.md

```diff
@@ -152,7 +152,7 @@ Docker from the repository.
 4.  Start Docker.
 
     ```bash
-    $ sudo systemctl docker start
```

Comment:
This part is going to be addressed in #1523, for all three affected distributions. If you remove that change from this PR, I'll merge this one. I checked and the problem in line 60 only affects Fedora. Thanks!